docs/installation.md

```diff
@@ -0,0 +1,103 @@
+# Installation
+
+## Deploy the Gateway
+
+You can deploy NGINX Kubernetes Gateway on an existing Kubernetes 1.16+ cluster. The following instructions walk through the steps for deploying on a [kind](https://kind.sigs.k8s.io/) cluster.
+
+1. Load the NGINX Kubernetes Gateway image onto your kind cluster:
+
+   ```
+   kind load docker-image nginx-kubernetes-gateway:0.0.1
+   ```
+
+   Make sure to substitute the image name with the name of the image you built.
+
+1. Install the Gateway CRDs:
+
+   ```
+   kubectl apply -k "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.5.0"
+   ```
+
+1. Create the nginx-gateway namespace:
+
+    ```
+    kubectl apply -f deploy/manifests/namespace.yaml
+    ```
+
+1. Create the njs-modules configmap:
+
+    ```
+    kubectl create configmap njs-modules --from-file=internal/nginx/modules/src/httpmatches.js -n nginx-gateway
+    ```
+
+1. Create the GatewayClass resource:
+
+    ```
+    kubectl apply -f deploy/manifests/gatewayclass.yaml
+    ```
+
+1. Deploy the NGINX Kubernetes Gateway:
+
+   Before deploying, make sure to update the Deployment spec in `nginx-gateway.yaml` to reference the image you built.
+
+   ```
+   kubectl apply -f deploy/manifests/nginx-gateway.yaml
+   ```
+
+1. Confirm the NGINX Kubernetes Gateway is running in `nginx-gateway` namespace:
+
+   ```
+   kubectl get pods -n nginx-gateway
+   NAME                             READY   STATUS    RESTARTS   AGE
+   nginx-gateway-5d4f4c7db7-xk2kq   2/2     Running   0          112s
+   ```
+
+## Expose NGINX Kubernetes Gateway
+
+You can gain access to NGINX Kubernetes Gateway by creating a `NodePort` Service or a `LoadBalancer` Service.
+
+### Create a NodePort Service
+
+Create a service with type `NodePort`:
```

Comment:
The lowercase style is already present across the docs and examples. Because of that, it is better to address that separately. Additionally, we can have style guide doc to capture rules like that.

Reply:
I don't think we need a separate guide outside of DocOps recommendations. An overall scrub would be outside this PR. I'll see if DocOps has anything published to this effect, I've had these discussions before though, just need to find where it's written down.